improver_tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -114,55 +243,53 @@ def set_up_cube(zero_point_indices=((0, 0, 7, 7),), num_time_points=1,
         zero_point_indices[index] = indices
     zero_point_indices = tuple(zero_point_indices)
 
-    data = np.ones((
-        num_realization_points, num_time_points,
-        num_grid_points, num_grid_points), dtype=np.float32)
+    data = np.ones(
+        (num_realization_points, num_time_points, num_grid_points, num_grid_points),
+        dtype=np.float32,
+    )
     for indices in zero_point_indices:
         realization_index, time_index, lat_index, lon_index = indices
         data[realization_index][time_index][lat_index][lon_index] = 0
 
-    cube = Cube(data, standard_name="precipitation_amount",
-                units="kg m^-2")
+    cube = Cube(data, standard_name="precipitation_amount", units="kg m^-2")
 
     cube.add_dim_coord(
-        DimCoord(
-            range(num_realization_points),
-            standard_name='realization'), 0)
+        DimCoord(range(num_realization_points), standard_name="realization"), 0
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [range built-in referenced when not iterating](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5354756)